### DISCUSSION TITLE
How much time

### DISCUSSION THREAD

**Discussion by QuickoAbdul** [mohankumarpaluru/oracle-freetier-instance-creation#21] [General]
Hello, once the script executes how long it can take for the instance to be created (give an approximate) and I wonder is it really a good idea to put a delay of 60 seconds, do we not risk get banned?

**Comment by QuickoAbdul** [comment]
hahhahahah it take like 10 minutes at 00AM

**Comment by mohankumarpaluru** [comment]
> Hello, once the script executes how long it can take for the instance to be created (give an approximate) 


It depends on the free instances in the region , sometimes immediate and sometimes it takes a long time (1 week was the longest it took for me personally, few others reported like it took even a month). I guess you are lucky. 

>  I wonder is it really a good idea to put a delay of 60 seconds, do we not risk get banned?

We are only trying to launch an instance through API oracle provided and not exploiting anything. Also oracle has some throttling on their side that prevents users from sending too many requests. So 60 secs should be fine.

**Reply by londondev77** [reply]
I’ve been running the script for months in the London region and no joy 🥹

**Reply by mohankumarpaluru** [reply]
May be stop and restart the script once, I'm not sure if it's due to the request oauth key change but sometimes stopping and starting helps

**Reply by londondev77** [reply]
Unfortunately, I've stopped and restarted it a number of times due to updating the script or when it's stopped for various reasons.

Could you confirm that the following is the expected output in case I have something misconfigured?:
`2024-06-17 09:09:38,708 - INFO - Command: launch_instance--
Output: {'status': 500, 'code': 'InternalError', 'message': 'Out of host capacity.'}
`

It's probably just down to capacity issues but just in case I have got the configuration wrong, I'll redo it from scratch.

**Reply by mohankumarpaluru** [reply]
this is the expected log

**Reply by londondev77** [reply]
Thanks. Guess London really has very little capacity. Will keep it running and keep my fingers crossed!
